opengrok-indexer/src/test/java/org/opengrok/indexer/util/TestRepository.java

```diff
@@ -53,6 +54,23 @@
     private static final String URL_FILE_PROTOCOL = "file";
     private static final char JAR_PATH_DELIMITER = '!';
 
+    private static final Map<Path, Path> renameMappings = new LinkedHashMap<>(Map.of(
+            Path.of("bazaar", "bzr"), Path.of("bazaar", ".bzr"),
+
+            Path.of("bitkeeper", "bk", "BitKeeper", "etc", "SCCS_dir"),
+            Path.of("bitkeeper", ".bk", "BitKeeper", "etc", "SCCS"),
+            Path.of("bitkeeper", "bk", "SCCS_dir"),
+            Path.of("bitkeeper", ".bk", "SCCS"),
+
+            Path.of("bitkeeper", "bk"), Path.of("bitkeeper", ".bk"),
```

Comment:
perhaps keep each `Path.of()` on separate line ?

Reply:
Sure, wanted to keep the key -> value mapping on the same line but some were just too long :D probably better to make it consistent.

Reply:
or insert empty line between each `Path.of()`